aws/resource_aws_lambda_function.go

```diff
@@ -385,12 +387,19 @@ func resourceAwsLambdaFunctionCreate(d *schema.ResourceData, meta interface{}) e
 func resourceAwsLambdaFunctionRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).lambdaconn
 
-	log.Printf("[DEBUG] Fetching Lambda Function: %s", d.Id())
-
 	params := &lambda.GetFunctionInput{
 		FunctionName: aws.String(d.Get("function_name").(string)),
 	}
 
+	// qualifier for lambda function data source
+	qualifier, qualifierExistance := d.GetOk("qualifier")
```

Comment:
Generally speaking, if there starts being logic like this in a shared resource/data source read function, we will want to not reuse the function.

Reply:
We can always adjust this later if necessary as its probably fine the way it currently is. 👍